src/gui/codeeditors/qgscodeeditorpython.cpp

```diff
@@ -348,6 +356,163 @@ void QgsCodeEditorPython::keyPressEvent( QKeyEvent *event )
   return QgsCodeEditor::keyPressEvent( event );
 }
 
+QString QgsCodeEditorPython::reformatCodeString( const QString &string )
+{
+  if ( !QgsPythonRunner::isValid() )
+  {
+    return string;
+  }
+
+  QgsSettings settings;
+  const QString formatter = settings.value( QStringLiteral( "pythonConsole/formatter" ), QStringLiteral( "autopep8" ) ).toString();
+  const int maxLineLength = settings.value( QStringLiteral( "pythonConsole/maxLineLength" ), 80 ).toInt();
```

Comment:
Maybe you could adress the issue raised in https://github.com/qgis/QGIS/pull/51733#issuecomment-1473400977 here? (Declare settings with the `QgsSettingsTree` API)

Reply:
Done